scripts/const.js

```diff
@@ -254,6 +262,11 @@ const ORACLE_CONTRACTS = {
         pyth: "0xA2aa501b19aff244D90cc15a4Cf739D2725B5729",
         ethusd: "0x6bF14CB0A831078629D993FDeBcB182b21A8774C",
     },
+    optimismSepolia: {
+        sequencer: "0x4200000000000000000000000000000000000011",
```

Comment:
I know this may had been confusing but rather than needing the actual contract of the sequencer, what we need is an oracle confirming the "liveliness" of the sequencer in the L2. 

Chainlink offers this. Check this section of their [docs: L2 sequencer feeds](https://docs.chain.link/data-feeds/l2-sequencer-feeds).